src/store/flowContext.test.ts

```diff
@@ -0,0 +1,225 @@
+import Localization from '../services/Localization';
```

Comment:
realizing now that i could have broken the reducers in `flowContext.ts`, `flowEditor.ts` and `nodeEditor.ts` into smaller reducers that are only concerned with a single key, e.g. `operand` in `nodeEditor.ts`, and then composed them w/ `combineReducers` (like the guy in that video did). do you think it makes sense to do so?